.github/workflows/cd.yml

```diff
@@ -0,0 +1,15 @@
+name: publish
```

Comment:
since you're removing the publish workflow, change the readme to include this one please